### DISCUSSION TITLE
AI-deck problem  Stuck at 99%

### DISCUSSION THREAD

**Discussion by coo122567** [bitcraze/discussions#449] [Q&A]
Hello everyone, 
what is the problem with the following picture ? 
![image](https://user-images.githubusercontent.com/118537645/205078774-8d854a04-a124-4d29-b5d9-cc5d87640c78.png)
![image](https://user-images.githubusercontent.com/118537645/205078859-35b05cef-0e33-4a6b-a17f-005254a27c5b.png)

**Comment by krichardsson** [comment]
Hi,
there seems to be a problem with the flashing functionality of the GAP8, please see https://github.com/bitcraze/aideck-gap8-examples/issues/104
